README.md

```diff
@@ -24,45 +24,64 @@ This tool was created to solve the problems mentioned above. It provides a new c
 
 The command uses the following conventions for managing static assets:
 
+#### Docs-Specific Static Assets
+
 - All static assets for the "next release" are at the root of the `docs/assets/` directory as a flat structure, that is, there are no sub-directories.
 - All static assets for "versioned docs" are at the root of the directory `docs/assets/version-<version-number>/` in a flat structure, that is, there are no sub-directories.
 
-    Ideally, the versioned assets would reside with the versioned documents (i.e. `website/versioned_docs/version-<version-number>/assets/`), but this cannot be achieved without changing the core logic of Docusaurus V1.
+  Ideally, the versioned assets would reside with the versioned documents (i.e. `website/versioned_docs/version-<version-number>/assets/`), but this cannot be achieved without changing the core logic of Docusaurus V1.
+
+
+#### Site-Specific Static Assets
+
+- All static assets are at the root of the `website/static` directory. Once a static asset type is versioned, a `next` directory for the "next release" and a `<version-number>` directory for the versioned files are generated.
```

Comment:
```suggestion
This refers to files in `website/static/` directory. Any directories at this location can be "versioned" if the path is specified. If a directory is versioned, it will be in the same location but under `<version-number>` and `next` (for next release) directories accordingly.

For example: if `website/static/dir1` directory is being versioned, files from that directory will be in following directories:

- `website/static/dir1/next`
- `website/static/dir1/<version-number>`

Any subsequent versioning will copy files from "next" release (`website/static/dir1/next`) into the new release (i.e. `website/static/dir1/<new-version-number>`

```